package-lock.json

```diff
@@ -0,0 +1,1465 @@
+{
```

Comment:
I think we want to prevent this file from being created and added in this project. See here for an example: https://github.com/airbnb/react-with-styles/commit/5175f997e40918509fd3fb6a2ba9f34311481eba

Reply:
whoops sorry!